### DISCUSSION TITLE
V2ray机场有人用吗？效果咋样

### DISCUSSION THREAD

**Discussion by quchengguo** [bannedbook/fanqiang#736] [Q&A]
V2ray机场有人用吗？效果咋样

**Comment by bannedbook** [comment]
好像有3元试用套餐，试试就知道了

**Reply by quchengguo** [reply]
节点偶尔断掉，得自己手动重新导入配置文件，相对麻烦了一些

**Comment by v2-free** [comment]
前几天墙疯狂发作，很多机场都出现一些中断，如果出现连接不上，请更新订阅更新节点即可，不需要重新导入。

**Comment by abeginner-h** [comment]
好像用不了了为才用了一天哇趣，第二天直接用不成明明还有1.5g流量，其他机场没问题
